src/test/java/com/wikia/webdriver/common/templates/AdsTestTemplate.java

```diff
@@ -0,0 +1,33 @@
+package com.wikia.webdriver.common.templates;
+
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+
+import java.lang.reflect.Method;
+
+/**
+ * @author drets
+ * @ownership AdEng
+ */
+public class AdsTestTemplate extends NewTestTemplateCore {
+
+  @BeforeClass(alwaysRun = true)
```

Comment:
I would like to get rid of logOut call since it will decrease the execution time of the tests. So, most obvious solution for me was creation a new test template or removing logOut call for all tests. I chose first one. :wink: Your suggestions how to get rid of logOut call?